compile error with nginx openresty 1.9.7

openresty compiles without mod_zip (latest github) but with the module, make fails with this error:

objs/src/http/ngx_http_postpone_filter_module.o:(.data+0x0): multiple definition of `ngx_http_postpone_filter_module'
objs/src/http/ngx_http_postpone_filter_module.o:(.data+0x0): first defined here
collect2: error: ld returned 1 exit status
objs/Makefile:362: recipe for target 'objs/nginx' failed
make[2]: *** [objs/nginx] Error 1
make[2]: Leaving directory '/root/openresty-1.9.7.3/build/nginx-1.9.7'
Makefile:8: recipe for target 'build' failed
make[1]: *** [build] Error 2
make[1]: Leaving directory '/root/openresty-1.9.7.3/build/nginx-1.9.7'
Makefile:4: recipe for target 'all' failed
make: *** [all] Error 2


compile options:

./configure --without-http_uwsgi_module --without-http_userid_module --without-http_autoindex_module --without-http_auth_basic_module --without-http_ssi_module --without-http_geo_module --with-pcre-jit --with-http_stub_status_module --with-http_mp4_module --add-module=/root/mod_zip

--- a/README.markdown
+++ b/README.markdown
@@ -15,8 +15,9 @@ To install, compile nginx with the following option:
 
     --add-module=/path/to/this/directory
 
-nginx 0.7.25 or later is required. If libiconv is present, support for the
-"X-Archive-Charset" header is enabled; see below.
+* nginx 0.7.25 or later is required
+* (optional) to enable the "X-Archive-Charset" header, libiconv is required
+* http_postpone must be enabled by including at least one of the http_addition, http_slice or http_ssi modules
 
 
 Usage

--- a/config
+++ b/config
@@ -5,13 +5,6 @@ NGX_ADDON_SRCS="$NGX_ADDON_SRCS $ngx_addon_dir/ngx_http_zip_parsers.c"
 NGX_ADDON_SRCS="$NGX_ADDON_SRCS $ngx_addon_dir/ngx_http_zip_file.c"
 NGX_ADDON_SRCS="$NGX_ADDON_SRCS $ngx_addon_dir/ngx_http_zip_headers.c"
 
-
-# This module depends upon the postpone filter being activated
-if [ $HTTP_POSTPONE != YES ]; then
-    HTTP_FILTER_MODULES="$HTTP_FILTER_MODULES $HTTP_POSTPONE_FILTER_MODULE"
-    HTTP_SRCS="$HTTP_SRCS $HTTP_POSTPONE_FILTER_SRCS"
-fi
-
 ngx_feature="iconv_open()"
 ngx_feature_name="NGX_ZIP_HAVE_ICONV"
 ngx_feature_run=no

